tests/test_views.py

```diff
@@ -139,6 +139,16 @@ def run_check(self):
         response = client.get(self.url, HTTP_ACCEPT='application/json')
         assert response['content-type'] == 'application/json'
 
+    def test_success_prefer_json(self, client):
```

Comment:
The more the merrier, but I believe we have that already covered, don't we?

Reply:
It is, but I don't think this particular case is quite there in the other tests.